packages/css/.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "rvest.vs-code-prettier-eslint",
```

Comment:
Do we have to use extensions both of these together? Or just using one of these is also OK?

Reply:
`rvest.vs-code-prettier-eslint` is an extension that uses ESLint for formatting.
- https://github.com/idahogurl/vs-code-prettier-eslint

Will it work just fine without it?
Since I copied your old settings, things may have changed.

Reply:
yeaa if our project is set with both of that, we should use all of it.
i just ask about this 'cause i just heard some projects got settings with just one of that.